ci-operator/step-registry/hypershift/kubevirt/install/disconnected/hypershift-kubevirt-install-disconnected-commands.sh

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "************ KubeVirt install disconnected command ************"
+
+function ocp_version() {
+    oc get clusterversion version -o jsonpath='{.status.desired.version}' | awk -F "." '{print $1"."$2}'
+}
+
+CNV_PRERELEASE_VERSION=${CNV_PRERELEASE_VERSION:-$(ocp_version)}
```

Comment:
We might need similar trick as here https://github.com/openshift/release/pull/48796/files#diff-679630ef070ddff84f07a0b118d953f9532a1fb4e60256fc392baaf79ee31920R15

To obtain the CNV image passed to prow job.

Reply:
OK, but let's do it in a follow-up and dedicated PR.